configs/mocha-config-compass/react.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+const base = require('./');
+
+module.exports = {
+  ...base,
+  require: base.require.concat([
+    'jsdom-global/register',
+    path.resolve(__dirname, 'chai-dom-register.js'),
```

Comment:
should we add also `sinon` and `sinon-chai` somewhere?

Reply:
Yep, good idea, seems like we use it a lot in tests suites (I somehow never noticed before)

Reply:
Done in a74c870